Assets/MRTK/StandardAssets/Shaders/MRTK.Shaders.sentinel

```diff
@@ -1 +1 @@
-ver: 2
\ No newline at end of file
+ver: 3
```

Comment:
Are we sure we should increment the sentinel if we're not modifying existing shaders, only adding a new one? I thought the sentinel is only for modifications that need to be overwritten onto any user's local changes?

Reply:
I am not sure, @Cameron-Micka is the authority on that question.

Reply:
@davidkline-ms wrote the sentinel file procedure. He would know better than I.

Reply:
I don't think we've shipped a release since the last increment (https://github.com/microsoft/MixedRealityToolkit-Unity/commit/fd8760ba32c3407cd1073425ac97916a53badbf7) so we can probably leave it for this change.

Reply:
I will decrease the version back to 2 then.